### ISSUE TITLE
Checking state of the PLUS button before activating Fast-Forward with a Wii Classic Controller?

### ISSUE THREAD

**Issue by bootsector** [ekeeke/Genesis-Plus-GX#231]
Hi there!

There are some SNES and PS1/PS2 adapter to Wiimote where you can hit START+SELECT in order to activate the HOME button.

The problem with most of those adapters and the current implementation of GenPlusGX Fast Forward key combo for a Wii Classic Controller is that if the user wants to open the menu interface, he would get Fast Forward activated instead. 

A quick fix would be checking the state of the PLUS/START button when checking the condition for the Fast Forward feature to kick in when using a Wii Classic Controller (MINUS + HOME - START).

**Comment by stubl** [comment]
speaking of fast forward key, I noticed when I started loading games from SD card instead of USB the fast forward key doesnt work anymore

is this an issue with SD?

**Comment by ekeeke** [comment]
>A quick fix would be checking the state of the PLUS/START button when checking the condition for the Fast Forward feature to kick in when using a Wii Classic Controller (MINUS + HOME - PLUS).

This solution is a little bit too much 'hackish' in my opinion as it is dedicated for this particular adapter (which always show HOME button pressed with MINUS and PLUS buttons pressed as well) and won't ever allow fast-forward to work with that adapter. A better (and not so hard to implement thanks to the way this is designed in the emulator) solution would be to make the fast-forward key combo configurable (just like the menu key combo) so you could remap itt to something else (HOME+R for example).

>is this an issue with SD?
Fast-forward works fine no matter where you load the game from, I exclusively use SD and never had any issue with it (and don't see any reason why the storage device would have any impact as it is totally unrelated to inputs and not used once the game is running, except in case of CD games off course).

**Comment by bootsector** [comment]
Agreed that my suggestion is really what we call here in Brazil as a "gambiarra".
I would love to see that configurable option!
Keep up the great work on this emulator! 
Cheers!
